srcs/juloo.keyboard2/KeyModifier.java

```diff
@@ -297,6 +300,32 @@ private static KeyValue handleFn(KeyValue k, int flags)
       case "c": name = "¢"; break;
       case "p": name = "₱"; break;
       case "€": case "£": return removed_key; // Avoid showing these twice
+      // alternating greek letters
+      case "π": name = "ϖ"; break;
+      case "θ": name = "ϑ"; break;
+      case "Θ": name = "ϴ"; break;
+      case "ε": name = "ϵ"; break;
+      case "β": name = "ϐ"; break;
+      case "ρ": name = "ϱ"; break;
+      case "σ": name = "ς"; break;
+      case "γ": name = "ɣ"; break;
+      case "φ": name = "ϕ"; break;
+      case "υ": name = "ϒ"; break;
+      case "κ": name = "ϰ"; break;
+      // alternating math characters
```

Comment:
While typing math, the Shift key might be used more often, to type uppercase greek letters/symbols. I think these symbols should be under `Shift` instead of `Fn` to be more discoverable.
The `Fn` would be a lot less useful on the greek pane, which is a good idea.

What do you think ?

Reply:
They are not uppercase but variations of the lowercase character. For example \phi is rendered as ϕ in LaTeX and \varphi as φ 
maybe that should be the other way around, but on the original Bone-Layout φ is the normal lowercase phi on Shift+M3+f and ϕ is on Shift+M3+q.
the same with:
\pi as π and \varpi as ϖ
\rho as ρ and \varrho as ϱ
etc.

On the desktop Bone-Layout these characters are on different keys (see Ebene 5 [here](https://www.neo-layout.org/Layouts/bone/)) but since we have fn as a general character alternating key here, I thought this would be the most obvious way to do it.